includes/class-geodir-api.php

```diff
@@ -214,7 +214,12 @@ public function register_wp_admin_settings() {
 			new GeoDir_Register_WP_Admin_Settings( $page, 'page' );
 		}
 	}
-	
+
+    /**
+     * Function for show in rest setup.
```

Comment:
"Function for show in rest setup." this is not a function, this is a class method, even if it were a function there is not need to mention it.